examples/angular/app.component.spec.ts

```diff
@@ -15,7 +16,7 @@ describe('AppComponent', () => {
   let fixture: ComponentFixture<AppComponent>;
   let app: AppComponent;
 
-  beforeEach(async(() => {
+  beforeEach(waitForAsync(() => {
```

Comment:
`async` is deprecated and replaced by `waitForAsync`. See https://angular.io/api/core/testing#functions